### PR TITLE
Updated Newtonsoft.Json to v12.0.2 and submodules. Fixed nuspec

### PR DESCRIPTION
**Pull request by Bykiev**
Updated Newtonsoft.Json to v12.0.2 and swagger-ui submodules. Fixed error in nuspec file (#84).

### CODE REVIEW

**Comment by Bykiev** [comment]
@heldersepu, what's wrong with AppVeyor?

**Comment by heldersepu** [comment]
```Submodule 'swagger-ui' (https://github.com/swagger-api/swagger-ui.git) registered for path 'swagger-ui'
Submodule 'swagger-ui-themes' (https://github.com/ostranme/swagger-ui-themes) registered for path 'swagger-ui-themes'
Cloning into 'C:/projects/swagger-net/swagger-ui'...
Cloning into 'C:/projects/swagger-net/swagger-ui-themes'...
Submodule path 'swagger-ui': checked out '9f10ab1d65eb2f2d3f1cb34e286e30775b21100d'
error: Server does not allow request for unadvertised object 327b9b3d8e931c7ec0e24f9bf560df3369524bc0
Fetched in submodule path 'swagger-ui-themes', but it did not contain 327b9b3d8e931c7ec0e24f9bf560df3369524bc0. Direct fetching of that commit failed.
Command exited with code 1```

Maybe don't change the submodules ...

File: Swagger.Net/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -19,5 +19,5 @@
 public struct SwaggerAssemb
 {
     public const string Title = @"Swagger.Net.Core";
-    public const string Version = @"8.3.22.202";
+    public const string Version = @"8.3.23";
```
Comment (heldersepu) [inline Swagger.Net/Properties/AssemblyInfo.cs]: Just bump the minor (8.3.22.203)

**Comment by heldersepu** [comment]
Updated test project:
http://swagger-net-test.azurewebsites.net/swagger/ui/index

**Comment by heldersepu** [comment]
`8.3.22.203` uploaded to NuGet:
https://www.nuget.org/packages/Swagger-Net/8.3.22.203

**Comment by Bykiev** [comment]
@heldersepu. #84 can be closed then

**Comment by narmontas** [comment]
Guys, updatig Newtonsoft dependency from major version 11 to 12 and changing only minor version for Swagger-Net itself was strange decision.

**Comment by Bykiev** [comment]
@narmontas, actually, Newtonsoft dependency update was introduced in 34d26fe4794aad28bcad65afccdb8d1b6a7976c0. Nuspec was outdated and it was fixed in this commit.

**Comment by narmontas** [comment]
Yes, but Nuspec dependency on Newtonsoft was not updated at that time. And now you fixed it without upping the major version.

**Comment by Bykiev** [comment]
@narmontas, in my point of view it doesn't matter, because with wrong Nuspec file it'll not working and will throw FileLoadException (see #84) if v11 of Newtonsoft is installed. So, the major version should be increased when it was actually updated.

**Comment by narmontas** [comment]
Nuspec for third-party library matters most.

1. We were already using Newtonsoft v11 from other dependencies so we added

```xml
<dependentAssembly>
  <assemblyIdentity name="Newtonsoft.Json" ... />
  <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="11.0.0.0" />
</dependentAssembly>
```

because your Nuspec was in disagreement with package reference.

2. Now you fixed Nuspec, but we cannot install your package minor update without forcing to ignore Newtonsoft dependency.

**Comment by Bykiev** [comment]
@narmontas, I see your trouble. Let's wait what @heldersepu think about it.

There are some possible workarounds:
1. Update Newtonsoft to latest version in project
2. Use AssemblyResolve
- Requires strong naming:
3. Use GAC
4. Use them side-by-side with codebase

**Comment by narmontas** [comment]
@Bykiev I already updated to latest version of the Newtonsoft. I just came here to see what was the reason behind this versioning misunderstanding. Nevertheless, doing great work with the Swagger-Net! :)

**Comment by heldersepu** [comment]
@narmontas not sure what you are suggesting...
instead of:
`8.3.22.203`
we should have done:
`9.3.22.20x`

The `3.22.2` I try to keep it matching the swagger-ui 
and just an update of a dependency I do not see it as minor change for us

**Pull request merged**